tests/CI/Docker/docker-compose/Dockerfile

```
@@ -0,0 +1,15 @@
+#
+#   Extension docker image to contain Docker-compose, curl and bash
+#       to run the tests scripts in.
+#
+#.......................................................................
+
+
+FROM docker:latest
+MAINTAINER "Anton Lu" anton.lu@cern.ch
+
+RUN apk add --update py-pip python-dev libffi-dev openssl-dev gcc libc-dev make curl bash
```

Required for the `git fetch --all`
```suggestion
RUN apk add --update py-pip python-dev libffi-dev openssl-dev gcc libc-dev make curl bash git
```